internal/webserver/server.go

```diff
@@ -230,12 +230,21 @@ func (webserver *WebServer) SetupTriggerRoute(handlerForTrigger func(http.Respon
 // StartHTTPServer starts the http server
 func (webserver *WebServer) StartHTTPServer(errChannel chan error) {
 	webserver.LoggingClient.Info(fmt.Sprintf("Starting HTTP Server on port :%d", webserver.Config.Service.Port))
```

Comment:
Please update to show protocol used instead of always `HTTP`

Reply:
done